packages/easy-coding-standard/src/ValueObject/Option.php

```diff
@@ -122,37 +123,35 @@ final class Option
     public const CONFIG = 'config';
 
     /**
-     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallelJobSize()
+     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallel()
      * @var string
      */
     public const PARALLEL_JOB_SIZE = 'parallel_job_size';
 
     /**
-     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallelPort()
      * @var string
      */
     public const PARALLEL_PORT = 'port';
 
     /**
-     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallelIdentifier()
      * @var string
      */
     public const PARALLEL_IDENTIFIER = 'identifier';
 
     /**
-     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallelMaxNumberOfProcesses()
+     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallel()
      * @var string
      */
-    public const PARALLEL_MAX_NUMBER_OF_PROCESSES = 'max-number-of-processes';
+    public const PARALLEL_MAX_NUMBER_OF_PROCESSES = 'max_number_of_processes';
 
     /**
      * @var string
      */
     public const MEMORY_LIMIT = 'memory-limit';
 
     /**
-     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallelTimeoutInSeconds()
+     * @see \Symplify\EasyCodingStandard\Config\ECSConfig::parallel()
      * @var string
      */
-    public const PARALLEL_TIMEOUT_IN_SECONDS = 'parallel-timeout-in-seconds';
+    public const PARALLEL_TIMEOUT_IN_SECONDS = 'parallel_timeout_in_seconds';
```

Comment:
The value should remain untouched, just in case it is used without constants. Changing value would break existing params.

Reply:
Got it. I have rollback